pkg/vault/contracts/authorizer/TimelockAuthorizerMigrator.sol

```diff
@@ -103,26 +126,68 @@ contract TimelockAuthorizerMigrator {
         vault.setAuthorizer(newAuthorizer);
     }
 
+    // Internal Functions
+
+    /**
+     * @notice Migrates to TimelockAuthorizer by setting up roles from the old Authorizer and new granters/revokers.
+     * @param n The number of permissions to set up on the new TimelockAuthorizer.
```

Comment:
Maybe document that n == MAX_UINT256 means migrate all, here and wherever "n" appears? Otherwise people might wonder what all the min's are doing.

Assuming we have this "n" business in the first place, allowing partial migrations, to ensure we don't run out of gas trying to do them all? (e.g., if there were 300 permissions.)